src/common/utils/tools.js

```diff
@@ -51,14 +51,33 @@ export const durationToSecond = duration => {
 
 export const isProduction = () => document.location.hostname.replace(/^www./, '') in AppIdMap;
 
-export const createUrl = (path, lang = getLanguage(), addLanguage = true, addHtmlExtension = true) => {
-    const pathBit = path ? `/${path}${addHtmlExtension ? '.html' : ''}` : '';
-    const languageBit = addLanguage && lang ? `/${lang}` : '';
-
+export const createUrl = (
+    urlSubdomain = false,
+    urlPath,
+    shouldAddLanguage = true,
+    shouldAddHtmlExtension = true,
+    isNonBotPage = true
```

Comment:
suggestion to make this an object instead, so it's clear what is being passed instead of depending on checking and remembering the order. it makes the function calls more readable as well.